srcpkgs/la-capitaine-icon-theme/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'la-capitaine-icon-theme'
+pkgname=la-capitaine-icon-theme
+version=0.6.1
+revision=1
+archs=noarch
+build_style=fetch
+short_desc="Icon pack designed to integrate with most desktop environments"
+maintainer="linarcx <linarcx@gmail.com>"
+license="GPL-3"
+homepage="https://github.com/keeferrourke/${pkgname}"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=9adb3123bf61464696400af43000b61de1825d2ce4b4ad4f21dd78091bd55e73
+
+do_install() {
+	tar xzf v${version}.tar.gz
```

Comment:
remove, it is automatic when build_style is not set

Reply:
@Chocimier  If i remove this line, and remove `build_style=` line, it gives me these errors:
https://gist.github.com/LinArcX/034e69d3ca0063d30c183ac3e5e2e729